src/d3a/models/strategy/predefined_pv.py

```diff
@@ -113,17 +116,24 @@ def _read_predefined_profile_for_pv(self):
             else:
                 self._power_profile_index = self.owner.config.cloud_coverage
         if self._power_profile_index == 0:  # 0:sunny
-            profile_path = pathlib.Path(d3a_path + '/resources/Solar_Curve_W_sunny.csv')
+            profile_path = (
+                pathlib.Path(d3a_path + "/resources/Solar_Curve_W_sunny_normalized.csv"))
         elif self._power_profile_index == 1:  # 1:partial
-            profile_path = pathlib.Path(d3a_path + '/resources/Solar_Curve_W_partial.csv')
+            profile_path = (
+                pathlib.Path(d3a_path + "/resources/Solar_Curve_W_partial_normalized.csv"))
         elif self._power_profile_index == 2:  # 2:cloudy
-            profile_path = pathlib.Path(d3a_path + '/resources/Solar_Curve_W_cloudy.csv')
+            profile_path = (
+                pathlib.Path(d3a_path + "/resources/Solar_Curve_W_cloudy_normalized.csv"))
         else:
             raise ValueError("Energy_profile has to be in [0,1,2,4]")
 
-        # Populate energy production forecast data
-        self.power_profile = read_arbitrary_profile(
-            InputProfileTypes.POWER, str(profile_path))
+        power_weight_profile = read_arbitrary_profile(
+            InputProfileTypes.IDENTITY, str(profile_path))
```

Comment:
Nit:
```suggestion
        power_weight_profile = read_arbitrary_profile(
            InputProfileTypes.IDENTITY, profile_path)
```
This check is already done here: https://github.com/gridsingularity/d3a-interface/blame/bfa8804079ec0edad2a6e5907fea526da3805eed/d3a_interface/read_user_profile.py#L232